serverless/messaging/api-cli/nats-cli.mdx

```diff
@@ -0,0 +1,111 @@
+---
+meta:
+  title: Using the NATS CLI
+  description: This page explains how to use the NATS CLI with Scaleway Messaging and Queuing
+content:
+  h1: Using the NATS CLI
+  paragraph: This page explains how to use the NATS CLI with Scaleway Messaging and Queuing
+tags: messaging nats cli
+categories: 
+  - messaging
+dates:
+  validation: 2023-01-20
+  posted: 2023-01-20
+---
+
+The NATS CLI (`nats`) is the official NATS tool for managing your NATS resources. It aloows you to simply create and manage your streams, consumers and more.
+
+Check out the official [NATS CLI documentation](https://docs.nats.io/using-nats/nats-tools/nats_cli) for installation instructions, examples and more.
+
+This page shows you how to get started with some basic actions via the NATS CLI.
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)  
+  - You have [created a NATS Messaging namespace](/serverless/messaging/how-to/create-namespace/)
+  - You have [generated credentials](/serverless/messaging/how-to/create-credentials/) for your NATS Messaging and Queuing namespace and downloaded them to your machine.
+</Message>
+
+## Installing the NATS CLI
+
+Follow instructions from the [official NATS documentation](https://docs.nats.io/using-nats/nats-tools/nats_cli#installing-nats). The installation process may differ depending on your OS.
+
+## Define a context
+
+To simplify your interactions with NATS hosted on Scaleway Messaging and Queuing, we recommend that you use [contexts](https://docs.nats.io/using-nats/nats-tools/nats_cli#nats-contexts). A context is a named configuration that stores the settings (such as credentials, URLs and certificates) required to connect to a NATS namespace. By creating a context, you won't need to specify your server and credentials with each new request.
+
+### Create a context
+
+The example below creates a context named `scaleway`.
+- Replace `{Scaleway NATS URL}` with the URL of your NATS namespace (find this in the console on your [Namespaces dashboard](https://console.scaleway.com/messaging/namespaces)).
+- Replace `{path to your creds file}` with the path to the location where you [downloaded your `.creds` file](https://www.scaleway.com/en/docs/serverless/messaging/how-to/create-credentials/#how-to-create-credentials-for-a-nats-namespace).
+
+```bash
+nats context save scaleway --server={Scaleway NATS URL} --creds={path to your creds file}
+```
+
+### Use the saved context
+
+Enter the following command to select the `scaleway` context for use:
+
+```
+nats context select scaleway
+```
+
+## Create a stream
+
+To create a [stream](/serverless/messaging/concepts/#stream), use the command `nats stream add` and follow the CLI guidelines.
```

Comment:
```suggestion
To create a [stream](/serverless/messaging/concepts/#stream/), use the command `nats stream add` and follow the CLI guidelines.
```

Reply:
I think the slash has to be in the middle (ie it has to be concepts/#<name-of-concept) but not at the end else it doesn't take you to the anchored concept, just the page